lite/examples/object_detection/raspberry_pi/README.md

```diff
@@ -142,3 +142,33 @@ You should see significantly faster inference speeds.
 For more information about creating and running TensorFlow Lite models with
 Coral devices, read [TensorFlow modles on the Edge TPU](
 https://coral.withgoogle.com/docs/edgetpu/models-intro/).
+
+
+
+# TensorFlow Lite Python object detection example with USB webcam.
+
+This is another example for Tensorflow lite with Raspberry Pi using USB webcam based on Pi camera version. In this example, you need to add one more from the settings that you performed on the Pi camera version. That is to install opencv.
```

Comment:
Curious if this example has to be marked as specific to the Raspberry Pi? shouldn't it be able to run on any GNU/Linux distro with OpenCV / Python?

Reply:
It doesn't need to be specific to raspberry pi, but I think it's appropriate to commit to this directory because tf-lite has a purpose to run on computer with limited resources.